mmv1/products/containerattached/api.yaml

```diff
@@ -0,0 +1,273 @@
+        output: true
+      - !ruby/object:Api::Type::String
+        name: createTime
+        description: |
+          Output only. The time at which this cluster was created.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: updateTime
+        description: |
+          The time at which this cluster was last updated.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: kubernetesVersion
+        description: |
+          The Kubernetes version of the cluster.
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: annotations
+        description: |
+          Optional. Annotations on the cluster. This field has the same
+          restrictions as Kubernetes annotations. The total size of all keys and
+          values combined is limited to 256k. Key can have 2 segments: prefix (optional)
+          and name (required), separated by a slash (/). Prefix must be a DNS subdomain.
+          Name must be 63 characters or less, begin and end with alphanumerics,
+          with dashes (-), underscores (_), dots (.), and alphanumerics between.
+      - !ruby/object:Api::Type::NestedObject
+        name: workloadIdentityConfig
+        description: |
+          Workload Identity settings.
+        output: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: identityProvider
+            description: |
+              The ID of the OIDC Identity Provider (IdP) associated to
+              the Workload Identity Pool.
+          - !ruby/object:Api::Type::String
+            name: issuerUri
+            description: |
+              The OIDC issuer URL for this cluster.
+          - !ruby/object:Api::Type::String
+            name: workloadPool
+            description: |
+              The Workload Identity Pool associated to the cluster.
+      - !ruby/object:Api::Type::NestedObject
+        name: loggingConfig
+        description: |
+          Logging configuration.
+        send_empty_value: true
+        allow_empty_object: true
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: componentConfig
+            description: |
+              The configuration of the logging components
+            send_empty_value: true
+            allow_empty_object: true
+            properties:
+            - !ruby/object:Api::Type::Array
+              name: enableComponents
+              description: |
+                The components to be enabled.
+              send_empty_value: true
+              allow_empty_object: true
+              item_type: !ruby/object:Api::Type::Enum
+                name: 'component'
+                description: |
+                  The components of the logging configuration.
+                values:
+                  - :SYSTEM_COMPONENTS
+                  - :WORKLOADS
+      - !ruby/object:Api::Type::NestedObject
+        name: errors
```

Comment:
I think this is inverted - it should be an array of objects where each object has a message key. Currently it would give output like this instead:

```
errors {
    message = [
        "error 1",
        "error 2",
        "error 3",
    ]
}
```

Reply:
Yeah, that makes sense!